Source/WebKit/UIProcess/Automation/WebAutomationSession.cpp

```diff
@@ -190,7 +190,7 @@ String WebAutomationSession::handleForWebPageProxy(const WebPageProxy& webPagePr
     if (iter != m_webPageHandleMap.end())
         return iter->value;
 
-    String handle = "page-" + createVersion4UUIDString().convertToASCIIUppercase();
+    String handle = makeString("page-", createVersion4UUIDString().convertToASCIIUppercase());
```

Comment:
If this was a performance critical code path, then I’d suggest creating an uppercasing concatenate adapter that does not allocate a `WTF::String`. We could either make one just for UUIDs or could create a generic, in-place, convert ASCII lowercase to uppercase adapter for use with StringConcatenate.

Reply:
I don't think it is performance critical code, so I leave it for now. It is nice if we have some way to enforce lower / upper cases.